lib/kubernetes-deploy/runner_task.rb

```diff
@@ -154,8 +154,8 @@ def set_container_overrides!(pod_definition, entrypoint, args, env_vars)
         raise TaskConfigurationError, message
       end
 
-      container.command = entrypoint
-      container.args = args
+      container.command = entrypoint if entrypoint
+      container.args = args if args
```

Comment:
Is this sufficient to make these optional?

Reply:
Seems like it.